test/framework/src/main/java/org/elasticsearch/search/NewlyReleasedQueryBuilder.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-package org.elasticsearch.search;
-
-import org.elasticsearch.Version;
-import org.elasticsearch.common.io.stream.StreamInput;
-import org.elasticsearch.xcontent.XContentParser;
-
-import java.io.IOException;
-
-/**
- * Query simulating a new named writable introduced in the current version.
- */
-public class NewlyReleasedQueryBuilder extends DummyQueryBuilder {
```

Comment:
are we no longer using this one?

Reply:
No, in the Rest layer tests I wanted to have both, in the transport layer we can simply overwrite the getMinumumRequiredVersion() function. The other dummy query is only necessary because I need something for the search template scripts that can be parsed back and throw some errors, but one of these is enough I think.